projects/js-packages/scan/composer.json

```
@@ -18,16 +18,12 @@
 			"pnpm run build"
 		],
 		"build-production": [
-			"NODE_ENV=production BABEL_ENV=production pnpm run build"
-		],
-		"watch": [
-			"Composer\\Config::disableProcessTimeout",
-			"pnpm run watch"
+			"pnpm run build"
```

I'd probably keep this as it was, even though the flags shouldn't make a difference anymore.
```suggestion
			"NODE_ENV=production BABEL_ENV=production pnpm run build"
```